server/server.js

```diff
@@ -0,0 +1,58 @@
+//**************************** SERVER *****************************
+const express = require('express');
+const server = express();
+const path = require('path');
+const port = process.env.PORT || 3010;
+
+const initialize = () => {
+    router.serveClient();
+    router.allowImageRetrieval();
+    server.listen(port, () => {
+        console.log(`listening on port ${port}`);
+    });
+}
+
+const router = {
+
+    serveClient: () => {
+        server.use(express.static('../client/dist'));
+        server.get('/', (req, res) => {
+            let index = path.join(__dirname, '../client/index.html');
+            res.sendFile(index, (err) => {
+                if (err) throw err;
+                res.end();
+            });
+        });
+    },
+
+    allowImageRetrieval: () => {
+        server.get('/images/:productID/', (req, res) => {
```

Comment:
I'm not totally sure that this conforms to the RESTful API standards; I think you would want something more along the lines of `/product/:productID/images` because you're getting all images associated with a certain productID. If you had a route that was`/images` and you send a GET request for that, you should get back all images in the db. Following the REST format you would want the next endpoint to be `/images/:imageID` to get just one image with a particular ID. The way you have it here seems to be combining two different routes. I think it's still totally fine, I'm just not sure if it follows the REST standards....I'm trying to find some material to confirm this and having trouble, so it might be a Josh question

Reply:
I guess to expand on this further, you wouldn't be able to get a specific image associated with specific product at this point like you would if your endpoint was `/products/:productId/images/:imageId`

Reply:
This is super helpful, thanks, I wasn't aware of the standards.  I'm reading about them here and I will make changes on the next revision:
https://docs.microsoft.com/en-us/azure/architecture/best-practices/api-design

Reply:
👏 
I've done a lot of API testing, so in my brain I was like "hm this doesn't seem right" but I was having a hard time justifying why! Glad to help